Atomic.md

```diff
@@ -0,0 +1,618 @@
+      BatchIndex: 1
+    },
+    Sequence: 0,
+    Fee: "0",
+    SigningPubKey: "",
+    TxnSignature: ""
+  }
+]
+```
+</details>
+
+# Appendix
+
+## Appendix A: FAQ
+
+### A.1: What if I want a more complex tree of transactions that are AND/XORed together? Can I nest `Atomic` transactions?
+
+The original version of this spec supported nesting `Atomic` transactions. However, upon further analysis, that was deemed a bit too complicated for an initial version of `Atomic`, as most of the benefit from this new feature does not require nested transactions. Based on user and community need, this could be added as part of a V2.
+
+### A.2: What if all of the transactions fail? Will a fee still be claimed?
+
+Yes, just as they would if they were individually submitted. 
+
+### A.3: Could there be an additional atomicity type for allowing all transactions to be processed regardless of whether they succeeded?
+
+This is unnecessary, since that is equivalent to submitting the transactions normally.
+
+### A.4: Would this feature enable greater frontrunning abilities?
+
+That is definitely a concern. Ways to mitigate this are still being investigated. Some potential answers:
+* Charge for more extensive path usage
+* Have higher fees for atomic transactions
+* Submit the atomic transactions at the end of the ledger
+
+### A.5: What error is returned if all the transactions fail in an `OR`/`BATCH` transaction?
+
+A general error, `temBATCH_FAILED`/`tecBATCH_FAILED`, will be returned. A list of all the return codes encountered for the transactions that were processed will be included in the metadata, for easier debugging.
+
+### A.6: Can another account sign/pay for the outer transaction if they don't have any of the inner transactions?
+
+If there are multiple parties in the inner transactions, yes. Otherwise, no. This is because in a single party `Atomic` transaction, the inner transaction's signoff is provided by `AtomicSigners`.
+
+### A.7: How is the `BATCH` atomicity type any different than existing behavior with sequence numbers?
+
+Right now, if you submit a series of transactions with consecutive sequence numbers without the use of tickets or `Atomic`, then if one fails in the middle, all subsequent transactions will also fail due to incorrect sequence numbers (since the one that failed would have the next sequence numbers).
+
+The difference between the `BATCH` atomicity type and this existing behavior is that right now, the subsequent transactions will only fail with a non-`tec` error code. If the failed transaction receives an error code starting with `tec`, then a fee is claimed and a sequence number is consumed, and the subsequent transactions will still be processed as usual.
+
+### A.8: Is it possible for inner transactions to end up in a different ledger than the outer transaction?
+
+No, because the inner transactions skip the transaction queue. They are already effectively processed by the queue via the outer transaction.
```

Comment:
This should also address how the inner transactions will be handled by consensus. I suggest that inner transactions be excluded from the proposal sets. Only the outer transaction is included, so - again - it's all or nothing.